tests/xDai/TWCoinTypeTests.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright © 2017-2021 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+//
+// This is a GENERATED FILE, changes made here MAY BE LOST.
+// Generated one-time (codegen/bin/cointests)
+//
+
+#include "../interface/TWTestUtilities.h"
+#include <TrustWalletCore/TWCoinTypeConfiguration.h>
+#include <gtest/gtest.h>
+
+
+TEST(TWxDaiCoinType, TWCoinType) {
+    auto symbol = WRAPS(TWCoinTypeConfigurationGetSymbol(TWCoinTypeXDai));
+    auto txId = WRAPS(TWStringCreateWithUTF8Bytes("0x9243890b844219accefd8798271052f5a056453ec18984a56e81c92921330d54"));
+    auto txUrl = WRAPS(TWCoinTypeConfigurationGetTransactionURL(TWCoinTypeXDai, txId.get()));
+    auto accId = WRAPS(TWStringCreateWithUTF8Bytes("0x12d61a95CF55e18D267C2F1AA67d8e42ae1368f8"));
+    auto accUrl = WRAPS(TWCoinTypeConfigurationGetAccountURL(TWCoinTypeXDai, accId.get()));
+    auto id = WRAPS(TWCoinTypeConfigurationGetID(TWCoinTypeXDai));
+    auto name = WRAPS(TWCoinTypeConfigurationGetName(TWCoinTypeXDai));
+
+    ASSERT_EQ(TWCoinTypeConfigurationGetDecimals(TWCoinTypeXDai), 18);
+    ASSERT_EQ(TWBlockchainEthereum, TWCoinTypeBlockchain(TWCoinTypeXDai));
+    ASSERT_EQ(0x0, TWCoinTypeP2shPrefix(TWCoinTypeXDai));
+    ASSERT_EQ(0x0, TWCoinTypeStaticPrefix(TWCoinTypeXDai));
+    assertStringsEqual(symbol, "xDAI");
+    assertStringsEqual(txUrl, "https://blockscout.com/xdai/mainnet/tx/0x9243890b844219accefd8798271052f5a056453ec18984a56e81c92921330d54");
```

Comment:
Note: this URL is not a valid tx explorer page, not found
https://blockscout.com/xdai/mainnet/tx/0x9243890b844219accefd8798271052f5a056453ec18984a56e81c92921330d54

Reply:
good catch